packages/datadog-instrumentations/src/bunyan.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const {
+  channel,
+  addHook
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+addHook({ name: 'bunyan', versions: ['>=1'] }, Logger => {
+  const logCh = channel('apm:bunyan:log')
+  shimmer.wrap(Logger.prototype, '_emit', emit => {
+    return function wrappedEmit (rec) {
+      if (logCh.hasSubscribers) {
+        logCh.publish({ logMessage: rec })
```

Comment:
This should only be `message` since the event is already `log` making it redundant.

Reply:
OK.